specs/notation-cli.md

```diff
@@ -1,284 +1,45 @@
-   --password value, -p value   Password for registry operations [$NOTATION_PASSWORD]
-   --help, -h                   show help (default: false)
-
-GLOBAL ARGUMENTS
-   --plain-http                 Registry access via plain HTTP (default: false)
-```
-
-## verify
-
-```console
-notation verify --help
-NAME:
-   notation verify - Verifies OCI Artifacts
-
-USAGE:
-   notation verify [command options] <reference>
-
-OPTIONS:
-   --signature value, -s value, -f value  signature files                     (accepts multiple inputs)
-   --cert value, -c value                 certificate names for verification  (accepts multiple inputs)
-   --cert-file value                      certificate files for verification  (accepts multiple inputs)
-   --pull                                 pull remote signatures before verification (default: true)
-   --local, -l                            reference is a local file (default: false)
-   --media-type value                     specify the media type of the manifest read from file or stdin (default: "application/vnd.docker.distribution.manifest.v2+json")
-   --username value, -u value             Username for registry operations [$NOTATION_USERNAME]
-   --password value, -p value             Password for registry operations [$NOTATION_PASSWORD]
-   --help, -h                             show help (default: false)
-
-GLOBAL ARGUMENTS
-   --plain-http                           Registry access via plain HTTP (default: false)
+This spec contains reference information on using notation commands. Each command has a reference page along with usages.
+
+## Notation Commands
+
+| Command                                     | Description                            |
+| ------------------------------------------- | -------------------------------------- |
+| [certificate](./commandline/certificate.md) | Manage certificates in trust store     |
+| [help](#notation-outline)                   | Provide a terse summary of all options |
+| [key](./commandline/key.md)                 | Manage keys used for signing           |
+| [list](./commandline/list.md)               | List signatures of the signed artifact |
+| [login](./commandline/login.md)             | Login to registries                    |
+| [logout](./commandline/logout.md)           | Log out from the logged in registries  |
+| [plugin](./commandline/plugin.md)           | Manage plugins                         |
+| [sign](./commandline/sign.md)               | Sign artifacts                         |
+| [verify](./commandline/verify.md)           | Verify artifacts                       |
+| [version](./commandline/version.md)         | Print the version of notation CLI      |
+
+## Notation Outline
+
+```text
+Notation - Notary V2 - a tool to sign, store, and verify artifacts
+
+Usage:
+  notation [command]
+
+Available Commands:
+  certificate Manage certificates in trust store
+  help        Provide a terse summary of all options
```

Comment:
This looks like we have a `help` command but in the above table, `help` points to `#notation-outline`. It doesn't make sense to have have `notation help --help` option.